react-todo/src/components/utils/dbPut.ts

```diff
@@ -0,0 +1,21 @@
+import { ITodoItem } from "../../Interfaces/Interfaces";
+
+export const dbPut = async (id: string, item: ITodoItem) => {
```

Comment:
Also, It's better to create post, put, get generic methods and call them with necessary params, url, etc